core/layout_api.php

```diff
@@ -940,7 +940,6 @@ function layout_sidebar_menu( $p_page, $p_title, $p_icon, $p_active_sidebar_page
 	echo '<i class="menu-icon fa ' . $p_icon . '"></i> ' . "\n";
 	echo '<span class="menu-text"> ' . lang_get_defaulted( $p_title ) . ' </span>' . "\n";
 	echo '</a>' . "\n";
-	echo '<b class="arrow"></b>' . "\n";
```

Comment:
Since this is a workaround, I think it would be best to comment-out the line instead of removing it, and add a comment with a @TODO note explaining why with a reference to the issue.

Reply:
We usually always remove unused code instead of commenting